### DISCUSSION TITLE
Marketing during the RSC transition

### DISCUSSION THREAD

**Discussion by matt-erhart** [solidjs/solid#1841] [Ideas]
Starting a greenfield, and I'm checking on all the frameworks. Wanted to share some impression and ideas:

I think solidjs's site should really show off features that it is especially good at and react is not so good, most of the examples I could find in a few minutes were things react is more than good enough for. Some things I assume solidjs would be much better at, and I think would be good to show up front to devs dropping by for a few minutes:
- figma like interactions
- large data grids
- https://www.npmjs.com/package/crossfilter2 dashboards
- anything drag and drop

The other frameworks are focus on sites, I assume solidjs could do sites and apps, but I think you should prove in examples we can see when dev drop by the main site. Good luck out there!

**Comment by samuelstroschein** [comment]
Agree! 

SolidJS = for highly interactive apps like Figma, Notion, etc, that need high performance and composability. 

React doesn't have the performance, Svelte doesn't have the composability. The competition against "site" frameworks like NextJS, Astro, & Co. is likely meaningless for enterprises because the other frameworks are good enough.

**Comment by matt-erhart** [comment]
Notion is a really good point. Text editing and syntax highlighting are probably the most common situations where we have a lot of html elements on screen and we want to update only one (and we really don't want to move into canvas/webgl). With all the AI text based apps, maybe solid could really shine in that space, and help folks create innovative UX.
